demisto_sdk/commands/common/content/objects/pack_objects/modeling_rule/modeling_rule.py

```diff
@@ -114,7 +114,7 @@ def fields(self):
         if not self._fields:
             uniq_fields = set(re.findall(self.RULE_FIELDS_REGEX, self.rule_text))
             uniq_fields.add(self.TIME_FIELD)  # The '_time' field is always required.
-            self.fields = uniq_fields
+            self.fields = sorted(uniq_fields)
```

Comment:
uniq_fields was set, after sorted it is list

Reply:
Let's also add typing to this function if we are alreday touching it 
`def fields(self) -> set:`